go_modules/lib/dependabot/go_modules/file_updater/go_mod_updater.rb

```diff
@@ -14,9 +14,15 @@ class GoModUpdater
         ENVIRONMENT = { "GOPRIVATE" => "*" }.freeze
 
         RESOLVABILITY_ERROR_REGEXES = [
+          # Private module could not be fetched
           /go: .*: git fetch .*: exit status 128/.freeze,
+          # The checksum in go.sum does not match the dowloaded content
           /verifying .*: checksum mismatch/.freeze,
+          # Private module could not be found during `go mod tidy`
+          /cannot find module providing package .*: exit status 128/m.freeze,
```

Comment:
Idea: could this be combined with the below to just `/cannot find module providing package/`?
It's maybe the same root, just being wrapped differently.

Reply:
Yeah we probably can, I was slightly worried about some other error also reusing that string and us matching on it by accident but the fact that you suggest it makes me feel that's probably not likely. Will update 👍

Reply:
My baseless assumption was this is something like `fmt.Errorf("cannot find module providing package: %s")` at the root. This seems [pretty close](https://github.com/golang/go/blob/ab2a5b48665eed6d670d719cdef5335bc3602359/src/cmd/go/internal/modload/import.go#L47) (and like the only hit for that string in the codebase?)

Then I assumed this was two different: `fmt.Errorf("build: %w", err)`  / `fmt.Errorf("tidy: %w", err)` paths wrapping - not sure where that's happening, but it seems 💯 that this error could be wrapped differently and should always be handled the same.